.github/workflows/build-and-publish.yml

```
@@ -22,6 +33,19 @@ jobs:
           dotnet-version: '9.0.x'
 
+      - name: Get current version
+        id: get-version
+        run: |
+          # Read version from project file
+          if [ -f "Bento/Bento.csproj" ]; then
+            CURRENT_VERSION=$(grep -oPm1 "(?<=<Version>)[^<]+" Bento/Bento.csproj || echo "1.0.0")
+          else
+            CURRENT_VERSION="1.0.0"
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)